examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+   c0    c1    c2
+r0 +-----+-----+-----+
+   |  1  |  2  |  2  |
+r1 +-----+-----+-----+
+   |  3  |  3  |  4  |
+   +-----+-----+-----+
```

Comment:
okay there is definitely some bug here. I get a 2x2 again. This one should definitely be as depicted above, where 2 and 3 are twice as wide as 1 and 4. It suggests the `stretch` parameters is either not set properly, or not respected properly during constraint solving, because 2 and 3 should have twice as much stretch / preferred width as 1 and 4.